onnxmltools/convert/sparkml/operator_converters/mlp_classifier.py

```diff
@@ -0,0 +1,101 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from pyspark.ml.classification import MultilayerPerceptronClassificationModel
+
+from ...common._registration import register_converter, register_shape_calculator
+from ...common.data_types import Int64TensorType, FloatTensorType
+from ...common.utils import check_input_and_output_numbers, check_input_and_output_types
+from ...common._topology import Operator, Scope, ModelComponentContainer
+from ....proto import onnx_proto
+from typing import List
+import numpy as np
+
+
+def convert_sparkml_mlp_classifier(scope: Scope, operator: Operator, container: ModelComponentContainer):
+    op: MultilayerPerceptronClassificationModel = operator.raw_operator
+    layers: List[int] = op.getLayers()
+    weights: np.ndarray = op.weights.toArray()
+
+    offset = 0
+
+    input: str
+    for i in range(len(layers) - 1):
+        weight_matrix = weights[offset : offset + layers[i] * layers[i + 1]].reshape([layers[i], layers[i + 1]])
+        offset += layers[i] * layers[i + 1]
+        bias_vector = weights[offset : offset + layers[i + 1]]
+        offset += layers[i + 1]
+
+        if i == 0:
+            input = operator.inputs[0].full_name
+
+        weight_variable = scope.get_unique_variable_name("w")
+        container.add_initializer(
+            weight_variable,
+            onnx_proto.TensorProto.FLOAT,
```

Comment:
A comment for future changes, it is possible to support double coefficients. It reduces the discrepancies.